src/other/PropertyEditor.js

```diff
@@ -129,11 +129,14 @@
         }
         if (widget) {
             var context = this;
-            this._watch = widget.monitor(function (paramId, newVal, oldVal) {
-                if (oldVal !== newVal) {
-                    context.lazyRender();
-                }
-            });
+            if(typeof widget.watch === "function"){
```

Comment:
"watch" has been replaced with "monitor" due to clash with Object.watch on FF - I suspect you may need to rebase your code?